shared/elements/gaia-header/dist/gaia-header.js

```diff
@@ -140,23 +140,12 @@ return w[n];},m.exports,m);w[n]=m.exports;};})('font-fit',this));
  */
```

Comment:
This will be updated in a separate commit once I got a r+ for https://github.com/gaia-components/gaia-header/pull/27

I include it here to make it easier to test.